test-fixtures/dedup-calibration-report.md

```
@@ -0,0 +1,271 @@
+| `cross_topic_sample` (gold distinct) | 15 | 0.426 | 0.572 | 0.670 |
+| `cross_topic_sample_extra` (gold distinct) | 13 | 0.424 | 0.578 | 0.646 |
+| `embedding_cluster_borderline` (kept as distinct) | 2 | 0.863 | 0.863 | 0.864 |
+| `human_curated_confirmed` (auto-distinct, kept after review) | 4 | 0.866 | 0.870 | 0.875 |
+
+---
+
+## 2. Empirical threshold analysis
+
+### 2.1 Distribution stats
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)